src/Symfony/Component/Mime/Tests/EmailTest.php

```diff
@@ -344,6 +344,12 @@ public function testGenerateBody()
         // 2 parts only, not 3 (text + embedded image once)
         $this->assertCount(2, $parts = $body->getParts());
         $this->assertStringMatchesFormat('html content <img src=3D"cid:%s@symfony">', $parts[0]->bodyToString());
+
+        $e = (new Email())->from('me@example.com')->to('you@example.com');
```

Comment:
would be better to create a new test instead of adding one more subtest into this test which does not respect the testing best practices. this has several advantages:

- the tests will run independently of other tests for the body generation (instead of running only when previous assertions of that god test have passed)
- the test will be named, which will make it clear what is being tested